prime-router/src/main/kotlin/Receiver.kt

```diff
@@ -247,6 +252,11 @@ open class Receiver(
                 }
             }
         }
+
+        if (topic != Topic.FULL_ELR && conditionFilter.isNotEmpty() && conditionFilter[0].isNotEmpty()) {
```

Comment:
This function overall seems kinda inefficient and confusing, but we've already checked the topic at line 236 so we probably can assume that it's not FULL_ELR here. I guess if that's likely to change, then it's good defensive programming to check again at this point, it's just hard for me to tell since I can't really understand how this function is intended to be used (It's currently only used in test cases 🤷 )

Reply:
So this function gets used when we call ./gradlew reload tables. I kept the FULL_ELR check since there is a todo labeling the above check as temporary.

Reply:
Good call on planning for the TODO, but I'm having a hard time verifying that this function is used in the gradle tasks, I can't seem to hit breakpoints in this function when I run reload tables or settings. I do see that it fails to reload settings when a covid receiver has a condition filter, but I'm still not seeing anything go through this function still. Any tips?

Reply:
For me, it only hit when I ran it from the command line. I think running it from IntelliJ kills the Attach to Container process because it can only run one at a time.

Reply:
Ah okay, that worked, thanks!